src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+
+@Entity
 public class Actor implements Cloneable {
```

Comment:
I think this model needs an `@Table(name = "actors")` annotation, so naming in DB would look better.
Every other model also have same inconvenience